### PR TITLE
Fix int tests

### PR DESCRIPTION
**Pull request by jimdickinson**
Integration tests needed a refactor after the last couple merged PRs.

- ginkgo_util doesn't have OperatorImage anymore
- Always import mage/config as cfgutil

### CODE REVIEW

File: tests/cluster_wide_install/cluster_wide_install_suite_test.go [hunk 1]
```diff
@@ -60,7 +61,7 @@ var _ = Describe(testName, func() {
 			//step := "setting up cass-operator resources via helm chart"
 
 			var overrides = map[string]string{
-				"image":              ginkgo_util.OperatorImage,
+				"image":              cfgutil.GetOperatorImage(),
```
Comment (johntrimble) [inline tests/cluster_wide_install/cluster_wide_install_suite_test.go:64]: Err, this one was my fault. I didn't see the `OperatorImage` references in these tests.

File: operator/pkg/reconciliation/reconcile_racks.go [hunk 2]
```diff
@@ -841,10 +857,15 @@ func (rc *ReconciliationContext) UpdateCassandraNodeStatus() error {
 			if nodeStatus.HostID == "" {
 				endpointsResponse, err := rc.NodeMgmtClient.CallMetadataEndpointsEndpoint(pod)
 				if err == nil {
+					useHostIP := rc.rpcAddressIsHostIP()
```
Comment (respringer) [inline operator/pkg/reconciliation/reconcile_racks.go:860]: Would we want to make a GetRpcAddress() that returns the HostIP or PodIP, whichever is configured?

Comment (jimdickinson) [reply operator/pkg/reconciliation/reconcile_racks.go]: great idea!

File: operator/pkg/reconciliation/reconcile_racks.go [hunk 3]
```diff
@@ -820,6 +820,22 @@ func findHostIdForIpFromEndpointsData(endpointsData []httphelper.EndpointState,
 	return ""
 }
 
+func getRpcAddress(dc *api.CassandraDatacenter, pod *corev1.Pod) string {
```
Comment (respringer) [inline operator/pkg/reconciliation/reconcile_racks.go:823]: awesome. +1

**Pull request merged**